.github/workflows/upload_coverage.yml

```diff
@@ -1,4 +1,4 @@
-name: Upload Code Coverage
```

Comment:
why is this called Upload, shouldnt it just be Code coverage

Reply:
The idea is "verb subject", but shorter names do seem preferable